src/libstd/lib.rs

```diff
@@ -12,33 +12,35 @@
 //!
 //! The Rust Standard Library provides the essential runtime
 //! functionality for building portable Rust software.
-//! It is linked to all Rust crates by default.
 //!
-//! ## Intrinsic types and operations
+//! It is linked to all Rust crates by default as though they
```

Comment:
It means 'in your crate root', which is how we usually say it. i'd update it to that wording.